lib/components/pagination-nav.vue

```diff
@@ -90,7 +90,7 @@
             </li>
             <li v-else class="page-item" role="none presentation">
                 <b-link class="page-link"
-                        v-bind="linkProps(currentPage - 1)"
+                        v-bind="linkProps(numberOfPages)"
```

Comment:
@mosinve I've now added an example using an array of links to generate the page links.